lib/index.js

```diff
@@ -51,7 +51,8 @@ var StandardValidationFilter = function (inputNode, options = {}) {
     format: options.format ? options.format : undefined,
     throwOnError: options.throwOnError ? options.throwOnError : false,
     logOutput: options.logOutput ? options.logOutput : false,
-    fix: options.fix ? options.fix : false
+    fix: options.fix ? options.fix : false,
+    globals: options.globals || options.global
```

Comment:
why accept both the singular and plural?

Reply:
To mimic standard accepting of both: https://github.com/standard/standard#i-use-a-library-that-pollutes-the-global-namespace-how-do-i-prevent-variable-is-not-defined-errors

Reply:
If you have both in your package.json do they use both to standard how do they handle that? are they merged? does one take precedence?

Reply:
It looks like `globals` takes precedence over `global`

https://github.com/standard/standard-engine/blob/master/index.js#L157